### ISSUE TITLE
Probabilities for classification task comparing to PyTorch

### ISSUE THREAD

**Issue by denisov-vlad** [ELS-RD/transformer-deploy#101]
Hi!

In default transformers I receive class probabilities with sum = 1, i.e. `label_0 = 0.7, label_1 = 0.3`.

When I convert the model via `transformer-deploy` to TensorRT and run it in Triton I receive different scores, i.e.: `['0.350586:0' '-0.162720:1']`

How they can be interpreted? 

Thanks!

**Comment by pommedeterresautee** [comment]
which model are you using ?
Can you provide the command line?

**Comment by denisov-vlad** [comment]
The model is based on BERT, detailed config is here: https://huggingface.co/vladenisov/sports-antihate/blob/main/config.json

I've used this command to convert model:
```
convert_model -m "vladenisov/sports-antihate" --backend tensorrt --seq-len 3 128 1024
```

**Comment by pommedeterresautee** [comment]
For instance:

```python
from transformers import BertForSequenceClassification, AutoTokenizer
import torch

model = BertForSequenceClassification.from_pretrained("vladenisov/sports-antihate")
model.eval()
tokenizer = AutoTokenizer.from_pretrained("vladenisov/sports-antihate")

with torch.inference_mode():
    input_ids = tokenizer.encode("something", return_tensors="pt")
    outputs = model(input_ids)
    print(outputs.logits)
    print(outputs[0].shape)
```

produces:

```log
tensor([[ 2.0995, -2.2072]])
torch.Size([1, 2])
```

So it seems to match your experience with `TensorRt`.
To get something which adds up to one you would need to perform a softmax on the output for instance but it won't change the perf of your model.

**Comment by denisov-vlad** [comment]
Sorry, my bad.

I've made prediction via transformers `pipeline()` method.

Thanks!
